include/maidsafe/crux/detail/multiplexer.hpp

```diff
@@ -398,6 +392,8 @@ void multiplexer::process_peek(boost::system::error_code error,
 {
     namespace asio = boost::asio;
 
+    is_receiving.store(false);
```

Comment:
Minor comment: This is equivalent to `is_receiving = false`, which may be easier to read.